### ISSUE TITLE
Remove heartbeat checking from E2E tests

### ISSUE THREAD

**Issue by wagmarcel** [Open-IoT-Service-Platform/platform-launcher#438]
The readiness of components are meanwhile checked with native K8s methods.
The old heartbeat tests before starting the E2E test is therefore no longer needed and will be removed from the respective services over time.
Therefore, this test needs to be removed from E2E tests.
